src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function cloneCell(row) {
+  const rowArr = Array.from(row.children);
+
+  const lastCell = rowArr[rowArr.length - 1];
+  const secondCellCopy = rowArr[1].cloneNode(true);
+
+  lastCell.before(secondCellCopy);
+}
+
+const tableArr = Array.from(document.querySelector('table').children);
+
+for (let i = 0; i < tableArr.length; i++) {
+  if (i === 1) {
+    const bodyRowsArr = Array.from(tableArr[i].children);
+
+    bodyRowsArr.forEach((row) => cloneCell(row));
+  } else {
+    const trNode = tableArr[i].firstElementChild;
+
+    cloneCell(trNode);
+  }
+}
```

Comment:
You can simplify your code like this
```suggestion
const table = document.querySelector('table');

for (const row of table.rows) {
  const copyColumn = row.cells[1].cloneNode(true);
  
  row.lastElementChild.before(copyColumn);
}
```

Reply:
That's waaaaaay simpler, indeed. Thanks and noted! I must've overlooked that useful link to documentation on table methods 😅